theoplayer/getting-started/01-sdks/03-ios/00-getting-started.mdx

```diff
@@ -0,0 +1,196 @@
+# Getting started on iOS/tvOS
+
+## Introduction
+
+This guide covers the steps to obtain and setup the core THEOplayer SDK targeting a native iOS/tvOS application. It also provides information and usage of the commonly used APIs.
+
+## Setup
+
+### THEOplayer license
+
+THEOplayer requires a valid THEOplayer license to be configured in order to access the SDK APIs (with the exception of some whitelisted sources hosted by our CDN). Request your license via [https://portal.theoplayer.com](https://portal.theoplayer.com).
+
+After creating your license, you can copy its license string to your clipboard as demonstrated in the screenshot below.
+
+![](https://cdn.theoplayer.com/images/git/theoplayer-ios-sdk-license-string.png)
+
+> **Note:** It's possible to add multiple platforms in a single license ie. iOS & tvOS.
```

Comment:
Try using an [admonition](https://docusaurus.io/docs/markdown-features/admonitions). 😉 

```suggestion
:::note

It's possible to add multiple platforms in a single license ie. iOS & tvOS.
 
:::
```
I see there's a few "Note:" blocks in this article, you might want to update them all.